pom.xml

```diff
@@ -83,6 +83,35 @@ under the License.
 
 	<build>
 		<plugins>
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>exec-maven-plugin</artifactId>
+				<version>3.1.0</version>
+				<inherited>false</inherited>
```

Comment:
Can we inherit any of this, plus config, from the parent pom?

Reply:
Resolved in offline discussion with Danny